combat-trainer.lic

```diff
@@ -3708,6 +3708,8 @@ class GameState
   def thrown_attack_verb
     if bound_weapon? && !@use_weak_attacks
       'hurl'
+    elsif attack_override
```

Comment:
❓ Why isn't `attack_override` the first check?

Reply:
I was being lazy. :D  fixed now.